transcom/mymove
# INT B-19883 add link to search to go to cust profile

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19883)

## Summary

We want the office users to have easy jobs and not confuse them with excessive clicks. With the addition of non-CAC users, we need the office user to be able to quickly search and have a quick icon to click to direct them to that user's customer information.

[B-19884](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19884) should handle the additional functionality for the office users to quickly validate customers as non-CAC users.

This PR does the following:
- adds a profile icon button that displays for both TOO and SC move searches, this directs them to the `CustomerInfo` page
- some enhancements to `CustomerInfo` component to eliminate a skinny screen by putting in a fixed width and flexbox
- adding conditional logic in the `handleClick` function when a user clicks a row after searching - this was a weird one. We have to override the function by checking if the user clicked the icon by assessing the parent div

### How to test

1. Access MM as an office user (TOO/SC)
2. Perform a move search for any user
3. Should see a profile icon display next to their name
4. Click that icon and should direct you to the Customer Info page

## Screenshots

![Screenshot 2024-05-21 at 12 29 33 PM](https://github.com/transcom/mymove/assets/136510600/ec1f5725-c99f-4028-88ff-fbb056b283bc)

TOO flow
https://github.com/transcom/mymove/assets/136510600/aa5ffc53-b42f-403d-9727-aca7705061e4

SC flow
https://github.com/transcom/mymove/assets/136510600/f25e8e69-3109-4db3-920b-d32d19afef47
